package/yast2-theme.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Oct  5 13:37:55 UTC 2017 - jreidinger@suse.com
+
+- add to SLE theme registration specific autoinstall.png
```

Comment:
It's Extensions/Modules specific, no registration in play :)

Reply:
what I mean here is that it is in yast2-registration used, which is not used in opensuse